.github/workflows/ci.yml

```diff
@@ -71,10 +71,13 @@ jobs:
         id: setup-ravendb
         shell: pwsh
         run: |
+          $hostInfo = curl -H Metadata:true "169.254.169.254/metadata/instance?api-version=2017-08-01" | ConvertFrom-Json
```

Comment:
What's this IP?

Reply:
The [Azure Instance Metadata Service](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/instance-metadata-service?tabs=windows)